### PR TITLE
Adding package seabird

### PR DESCRIPTION
**Pull request by castelao**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: seabird/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name = "seabird" %}
```
Comment (synapticarbors) [inline seabird/meta.yaml:1]: this recipe needs to be in `recipes/seabird/meta.yaml`

Comment (castelao) [reply seabird/meta.yaml]: Ouch! Where is my mind? I'm sorry about that.

File: recipes/seabird/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - click >=6.6
+    - numpy >=1.14
+    - setuptools >=49.1
```
Comment (scopatz) [inline recipes/seabird/meta.yaml:26]: Do you need setuptools at runtime?

Comment (synapticarbors) [reply recipes/seabird/meta.yaml:26]: uses `pkg_resources`, so I think so

**Comment by synapticarbors** [comment]
Linting error is a bug, so everything looks good.

**Pull request merged**